packages/circuit-ui/components/Toggletip/Toggletip.tsx

```diff
@@ -63,7 +62,9 @@ export interface ToggletipProps
     | 'animationDuration'
     | 'initialFocusRef'
     | 'open'
-    | 'preventClose'
+    | 'preventOutsideClickClose'
+    | 'preventEscapeKeyClose'
+    | 'hideCloseButton'
```

Comment:
Suggestion: We could export a `PublicDialogProps` type from the Dialog component to avoid having to omit all these private props in every component that builds on top of it (and potentially forgetting to omit one in the future).